components/style/properties/helpers.mako.rs

```diff
@@ -145,6 +151,19 @@
                     }).map(SpecifiedValue)
                 % endif
             }
+
+            % if requires_single:
```

Comment:
We should get rid of this and refactor code using these functions to use the `single_value` functions instead